apps/codeforafrica/src/payload/fields/links/link.js

```diff
@@ -31,6 +31,7 @@ const link = ({
   disableOpenInNewTab = false,
   overrides = {},
   required = true,
+  labelFields: fields = [],
```

Comment:
Why do we need this?

Reply:
@kilemensi we need extra field for icons in linkArray

Reply:
We can override linkArray fields directly